Scheduler add getting started (#1484)
DevExpress/devextreme-documentation: concepts/05 UI Components/Scheduler/00 Getting Started with Scheduler/15 Edit Appointments.md

> Users can do the following actions on appointments:

Reviewer: "Do" is obviously a weak verb, but I guess this is an easy way to hide the word "perform". I think the "manage appointments" phrase you wanted to use in the beginning is more appropriate here: 

```suggestion
Users can manage appointments in the following ways:
```

* Add new appointments

* Update appointments in an edit form

* Resize an appointment to change its duration

* Drag and drop an appointment to reschedule it

* Remove appointments

> Edit operations are enabled by default. To disable an individual operation, assign **false** to its property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.

Reviewer: ```suggestion
Edit operations are enabled by default. To disable an individual operation, assign **false** to the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.
```

> Edit operations are enabled by default. To disable an individual operation, assign **false** to the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.

Reviewer 2: 1) `Edit operations are enabled by default.`
We need to rewrite this without using "by default". For example, can we say the following?

`Edit operations are enabled (the default setting).`

Or do you mean:

`Edit operations are initially enabled.`

2) `To disable an individual operation, assign **false** to the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object.`
Can we change this to the following?

`To disable an individual operation, set the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object to **false**.`

```suggestion
Edit operations are enabled (the default setting). To disable an individual operation, set the corresponding property in the [editing](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/) object to **false**. In this tutorial, we use the [allowDragging](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/editing/#allowDragging) property to disable appointment drag-and-drop.
```
